lib/generators/avo/resource_generator.rb

```diff
@@ -185,14 +189,28 @@ def fields_from_model_tags
 
         def fields_from_model_associations
           associations.each do |name, association|
-            fields[name] = if association.is_a? ActiveRecord::Reflection::ThroughReflection
-              field_from_through_association(association)
-            else
-              ::Avo::Mappings::ASSOCIATIONS_MAPPING[association.class]
-            end
+            fields[name] =
+              if association.polymorphic?
+                types = find_polymorphic_association_types(association)
+                {field: "polymorphic", options: {types: types}}
```

Comment:
Can we use `field: "belongs_to"` and add the `polymorphic_as` inside the `options`?